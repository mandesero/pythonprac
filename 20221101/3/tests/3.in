f, g, h = Grange(1111, 1111, 1048575), Grange(222, 222, 1048576), Grange(333, 333, 1048577)
print(f"{f=}, {g=}, {h=}")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(h))=}")
print(f"{f[0]=}, {h[24]=}")
a, b = f[3:327], h[3:367:2]
print(f"{a=}, {a[5]=}")
print(f"{b=}, {b[5]=}, {list(b)=}")
