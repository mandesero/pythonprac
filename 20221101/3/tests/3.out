f=grange(1111, 1111, 1048575), g=grange(222, 222, 1048576), h=grange(333, 333, 1048577)
len(f)=1, len(g)=2, len(h)=2
len(list(f))=1, len(list(g))=2, len(list(h))=2
f[0]=1111, h[24]=1151080873420685623137147071285646130496476491735729728327067693
a=grange(3, 1111, 327), a[5]=5077986587359653
b=grange(3, 110889, 367), b[5]=50299490941916791641520347, list(b)=[3]
